### ISSUE TITLE
Facebook shop is being processed-help

### ISSUE THREAD

**Issue by Ethnichoice** [facebookincubator/facebook-for-woocommerce#366]
I have added my WooCommerce site to my facebook business page. I am still getting the message "Your shop is being processed
Once processing is complete, your shop will be visible to people who visit your Page. You can still create and run ads promoting your products on Facebook."

My facebook page url  https://www.facebook.com/ethnichoice/
My facebook shop page url https://www.facebook.com/ethnichoice/shop/
My webpage's shop page url https://ethnichoice.com/shop/

Please Help.Thanks!

**Comment by jaredkotoff** [comment]
Im having the same issue. My shop has been processing for over a month.

**Comment by dmitridr** [comment]
Hello,

I have some good news here, we've identified the problem (it was a problem in our review system) and we have a fix for newly created shops going live in a few hours. We plan to fix all *existing* shops that are having this problem by end of day tomorrow. Will update more when we've fixed everything. Thanks for your patience!

**Comment by dmitridr** [comment]
Hey again,

Happy to say that we flat out fixed the problem for 95% of people with this issue, so with luck both your shops should be live now. (@Ethnichoice yours definitely is!)

If you're in the 5% we could not fix, try the following, first wait ~2-3 hours (it may still be in the review system because people are reviewing it). If it's still not working, reset the plugin using `Reconfigure > Advanced Options > Delete` then wait 48 hours, the shop should be processed correctly.

If that *still* does not work, which is very unlikely, please let us know by opening a new issue. That said, most likely everything is working for you now!

**Comment by krregg** [comment]
> Hey again,
> 
> Happy to say that we flat out fixed the problem for 95% of people with this issue, so with luck both your shops should be live now. (@Ethnichoice yours definitely is!)
> 
> If you're in the 5% we could not fix, try the following, first wait ~2-3 hours (it may still be in the review system because people are reviewing it). If it's still not working, reset the plugin using `Reconfigure > Advanced Options > Delete` and then complete the setup again, then wait 48 hours, the shop should be processed correctly.
> 
> If that _still_ does not work, which is very unlikely, please let us know by opening a new issue. That said, most likely everything is working for you now!

Hi Dmitridr,
this is an old post but anyway, my web shop is being in process for a few days. Business is running I have to pay bills but I still can't sell anything trough the Facebook. Can you please help me somehow? My page is: https://www.facebook.com/forever420official/
If you can't help me, please let me know where I can find help.

Thank you in advance!
